### ISSUE TITLE
update

### ISSUE THREAD

**Issue by jonathanong** [jshttp/style-guide#30]
@Fishrock123 not 100% sure what our new style guide is, but i'm pretty sure this is out of date and could be deleted

**Comment by Fishrock123** [comment]
Hmmmm, why would we have a new style guide?

Also what is outdated about this?

**Comment by Fishrock123** [comment]
(But we should probably just use eslint or standard)

**Comment by jonathanong** [comment]
i'm assuming node has one that we should follow, no? would make things simpler

yeah, i'm :+1: with standard or airbnb or whatever as long as there is one. `eslint --fix` is awesome

**Comment by Fishrock123** [comment]
Nah, node core's styling is very specific to it, mostly so that we don't need to go and change the entire codebase. I don't think we should use the exact same thing.
